packages/api/portfolioDrafts/application/createApplicationStep.ts

```diff
@@ -0,0 +1,64 @@
+import { UpdateCommand } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { ApplicationStep } from "../../models/ApplicationStep";
+import { APPLICATION_STEP } from "../../models/PortfolioDraft";
+import { dynamodbClient as client } from "../../utils/dynamodb";
+import { DATABASE_ERROR, NO_SUCH_PORTFOLIO_DRAFT, REQUEST_BODY_EMPTY, REQUEST_BODY_INVALID } from "../../utils/errors";
+import { ApiSuccessResponse, SuccessStatusCode } from "../../utils/response";
+import { isApplicationStep, isValidJson } from "../../utils/validation";
+
+/**
+ * Submits the Application Step of the Portfolio Draft Wizard
+ *
+ * @param event - The POST request from API Gateway
+ */
+export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> {
+  if (!event.body) {
+    return REQUEST_BODY_EMPTY;
+  }
+
+  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
+
+  if (!portfolioDraftId) {
+    return NO_SUCH_PORTFOLIO_DRAFT;
+  }
+
+  if (!isValidJson(event.body)) {
+    return REQUEST_BODY_INVALID;
+  }
+  const requestBody = JSON.parse(event.body);
+
+  // remove this, and function?
+  if (!isApplicationStep(requestBody)) {
+    return REQUEST_BODY_INVALID;
+  }
```

Comment:
Remove in light of Zach's schema validation work?  Or leave for now?

Reply:
I would leave it for now.